File: dbflow-tests/src/androidTest/java/com/raizlabs/android/dbflow/test/customseparator/CustomSeparatorClass.java
package com.raizlabs.android.dbflow.test.customseparator;

import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

/**
 * Description:
 */
@Table(database = CustomSeparatorDatabase.class)
public class CustomSeparatorClass extends BaseModel {

    @PrimaryKey
    int id;
}

File: dbflow-tests/src/test/java/com/raizlabs/android/dbflow/customseparator/CustomSeparatorClass.kt
package com.raizlabs.android.dbflow.customseparator

import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

/**
 * Description:
 */
@Table(database = CustomSeparatorDatabase::class)
class CustomSeparatorClass : BaseModel() {

    @PrimaryKey
    var id: Int = 0
}
